src/components/MarketingBanner/MarketingBanner.tsx

```diff
@@ -1,15 +1,21 @@
-import React, { FC, useCallback, useEffect, useState } from 'react';
+import React, { useCallback, useEffect, useState } from 'react';
 import { useOktaAuth } from '@okta/okta-react';
 import { useSelector } from 'react-redux';
 import { getAuth } from 'store/auth';
 import { displayAndLogError, useStyles } from 'core';
 import { QueryStatus } from '@reduxjs/toolkit/dist/query';
 import { LoadingButton } from '@mui/lab';
+import { UserClaims } from '@okta/okta-auth-js';
 import { getStyles } from './MarketingBanner.styles';
 import { Messages } from './MarketingBanner.messages';
 import { useEditProfileMutation } from '../../core/api/auth.service';
 
-export const MarketingBanner: FC = ({ children }) => {
+interface IMarketingBanner {
```

Comment:
```suggestion
interface MarketingBannerProps {
```

Please don't prepend interfaces with `I`